lib/package-manager/get-binary.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
The name of this file is inaccurate, since neither `npm` nor `yarn` are binaries (Regardless that we have the `./bin` conventions).
But IMO the name should be `get-executable.js`